src/System.Diagnostics.EventLog/tests/Configurations.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp2.0-Windows_NT;
+      netcoreapp-Windows_NT;
```

Comment:
Are we losing test coverage by doing this?

Reply:
We aren't losing any coverage that we don't have gaps in all over corefx. We generally only test the latest builds for each platform because we don't have a great system to verify all possible configurations. Our strategy is to use the latest versions as a proxy for the other configurations.